Проект 6 спринта POM

ba-polkov opened the pull request:
Выполненный проект 6 спринта

ba-polkov committed:
Проект 6 спринта POM

Выполненный проект 6 спринта

DolgoarshinnykhLG commented on config.py:
```diff
@@ -0,0 +1,4 @@
+BASE_URL = 'https://qa-scooter.praktikum-services.ru/'
```
Можно лучше здесь и далее: адреса лучше вынести во внешний модуль(например, urls). Они могут измениться, что затруднит поддержку тестов

DolgoarshinnykhLG commented on page/order_page.py:
```diff
@@ -0,0 +1,52 @@
+    
+    @allure.step ('Ввод даты в поле ввода')
+    def send_keys_date_by_keyboard_input(self):
+        self.send_keys_to_input(OrderPageLocators.input_date).send_keys(TestData.test_data_user_1[5])
```
Необходимо исправить здесь и далее: метод этого класса должен получать тестовые данные в качестве аргумента из тела теста

ba-polkov committed:
Исправления

1. Создан модуль url для переменной BASE_URL для оптимизации поддержки тестов
2. В метод send_keys_date_by_keyboard_input передан аргумент test_data (тестовые данные) для возможности дальнейшего переиспользования
3. Исправлены опечатки в наименовании метода wait_visibility_of_element
